components/gantry/oneaxis/oneaxis.go

```diff
@@ -439,6 +199,7 @@ func (g *oneAxis) MoveToPosition(
 func (g *oneAxis) Stop(ctx context.Context, extra map[string]interface{}) error {
 	ctx, done := g.opMgr.New(ctx)
 	defer done()
+	g.opMgr.CancelRunning(ctx)
 	return g.motor.Stop(ctx, extra)
```

Comment:
Either lock the mutex in this function, or add a comment explaining that locking the mutex here will cause us to block until all movement is complete, thus defeating the purpose of `Stop` entirely.

If the latter, what happens if some other goroutine tries to interact with the motor after we've canceled the opMgr but before we've stopped the motor?

Reply:
I actually changed Stop back. motor's Stop implements the cancel Running anyway.